test/extended/image_ecosystem/plugin.go

```diff
@@ -414,18 +417,46 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 		g.By("kick off the build for the jenkins ephermeral and application templates")
 		tag := []string{"openshift/jenkins-plugin-snapshot-test:latest"}
 		hexIDs, err := exutil.DumpAndReturnTagging(tag)
-		var jenkinsEphemeralPath string
-		var testingSnapshot bool
-		if len(hexIDs) > 0 && err == nil {
-			// found an openshift pipeline plugin test image, must be testing a proposed change to the plugin
-			jenkinsEphemeralPath = exutil.FixturePath("testdata", "jenkins-ephemeral-template-test-new-plugin.json")
-			testingSnapshot = true
+
+		// If the user has expressed an interest in local plugin testing by setting the
+		// SNAPSHOT_JENKINS_IMAGE environment variable, try to use the local image. Inform them
+		// either about which image is being used in case their test fails.
+		snapshotImagePresent := len(hexIDs) > 0 && err == nil
+		useSnapshotImage := os.Getenv(useLocalPluginSnapshotEnvVarName) != "" && snapshotImagePresent
+
+		//TODO disabling oauth until we can update getAdminPassword path to handle oauth (perhaps borrow from oauth integration tests)
+		newAppArgs := []string{exutil.FixturePath("testdata", "jenkins-ephemeral-template-no-oauth.json")}
+
+		if useSnapshotImage {
+			ginkgolog("")
+			ginkgolog("")
+			ginkgolog("IMPORTANT: You are testing a local jenkins snapshot image.")
+			ginkgolog("In order to target the standard image stream, you must unset %s before running extended tests.", useLocalPluginSnapshotEnvVarName)
+			ginkgolog("")
+			ginkgolog("")
+
+			// Create an imagestream based on the Jenkins' plugin PR-Testing image (https://github.com/openshift/jenkins-plugin/blob/master/PR-Testing/README).
+			err = oc.Run("new-build").Args("-D", "FROM openshift/jenkins-plugin-snapshot-test", "--to", "jenkins-plugin-snapshot-test").Execute()
```

Comment:
make the FROM image name a constant and use the same constant on line 418 (the "tag" array).

i don't think there should be any issue with having ":latest" on the image name in both cases.